firefox-ios/Client/Application/ImageIdentifiers.swift

```diff
@@ -70,5 +70,6 @@ public struct ImageIdentifiers {
         public static let socialMediaTrackers = "socialMediaTrackersImage"
         public static let fingerprintersTrackers = "fingerprintersTrackersImage"
         public static let crossSiteTrackers = "crossSiteTrackersImage"
+        public static let analyticsTrackersImage = "analyticsTrackersImage"
```

Comment:
this image looks like it lives in our icon library, should we move these to `StandardImageIdentifiers` instead?

**Figma File**: https://www.figma.com/design/Kv7wWr8vH6AQrGhu9vmjdE/Mobile-Icons?node-id=2129-1787&t=z2Hcgkam8gJpbCn8-0
**Github repo**: https://github.com/FirefoxUX/acorn-icons/tree/main/icons/mobile/24/pdf
cc: @thatswinnie